stack-easybib/recipes/role-gearmanw.rb

```diff
@@ -1,9 +1,8 @@
 include_recipe 'stack-easybib::role-phpapp'
 
-include_recipe 'php::module-mysqli'
```

Comment:
Trying to think if the `php::module-x` should be kept instead and you basically just have to set the `package_prefix` if necessary.